workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/CodeGeneratorOptions.cs

```diff
@@ -53,6 +56,18 @@ public static CodeGeneratorOptions ParseArguments(ICollection<string> args)
                     "serialization-override=", "OPTIONAL: defines an override for serialization of a single type",
                     u => options.SerializationOverrides.Add(u)
                 },
+                {
+                    "disable-verbose-logging=", "OPTIONAL: disable verbose logging",
+                    v => options.DisableVerboseLogging = string.IsNullOrEmpty(v) || v.Equals("true")
```

Comment:
Why not make this a toggle rather than accept a value? Then you can structure it the same as the `help` flag

https://github.com/spatialos/gdk-for-unity/pull/1227/files#diff-8db6365359413d2c0c9a382513957a19R72